pkg/kubelet/kuberuntime/labels.go

```diff
@@ -56,6 +60,7 @@ type annotatedPodSandboxInfo struct {
 
 type labeledContainerInfo struct {
 	ContainerName string
+	ContainerType string
```

Comment:
can we create a separate ContainerType type here?
`type ContainerType string`
`const ContainerType containerTypeInit = "INIT"`
...